src/java/org/apache/cassandra/net/Verb.java

```diff
@@ -259,6 +288,13 @@
     ACCORD_GET_DEPS_RSP         (148, P2, writeTimeout, REQUEST_RESPONSE, () -> GetDepsSerializers.reply, RESPONSE_HANDLER),
     ACCORD_GET_DEPS_REQ         (147, P2, writeTimeout, ACCORD,               () -> GetDepsSerializers.request,       () -> AccordService.instance().verbHandler(), ACCORD_GET_DEPS_RSP),
 
+    CONSENSUS_KEY_MIGRATION_FINISHED(149, P1, writeTimeout, MUTATION, () -> ConsensusKeyMigrationFinished.serializer, () -> ConsensusKeyMigrationState.consensusKeyMigrationFinishedHandler),
+
+    UPDATE_CM_RSP(151, P1, writeTimeout, REQUEST_RESPONSE, () -> NoPayload.serializer, () -> ResponseVerbHandler.instance),
```

Comment:
These are definitely going away once we merge with TCM since they are just a fake version of it. I'll rename them, but the length of the verbs is an issue if we are going to try and also get the formatting to match. `CONSENSUS_KEY_MIGRATION_FINISHED` isn't fitting, and I can either make it shorter or I can push out all the other lines to match. WDYT?

Reply:
Since UPDATE_CM* is going away one we have TrM I left it alone.